module/Application/view/error/404.phtml

```diff
@@ -1,5 +1,13 @@
 <?php
+
+/**
+ * @see       https://github.com/laminas/laminas-mvc-skeleton for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-mvc-skeleton/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-mvc-skeleton/blob/master/LICENSE.md New BSD License
+ */
+
```

Comment:
The doc blocks in view scripts are the first lines a user will delete, so we omit them.